BookPlayer/Hardcover/BookPicker/HardcoverBookPickerView.swift

```diff
@@ -0,0 +1,147 @@
+//
+//  HardcoverBookPickerView.swift
+//  BookPlayer
+//
+//  Created by Jeremy Grenier on 6/28/25.
+//  Copyright © 2025 BookPlayer LLC. All rights reserved.
+//
+
+import BookPlayerKit
+import SwiftUI
+
+struct HardcoverBookPickerView: View {
+  @Environment(\.dismiss) var dismiss
+
+  @ObservedObject var viewModel: HardcoverBookPickerView.Model
+
+  var body: some View {
+      Group {
+        switch viewModel.loading {
+        case .fetching:
+          VStack(spacing: 16) {
+            ProgressView()
+            Text("hardcover_searching_books".localized)
+              .foregroundColor(.secondary)
+          }
+          .frame(maxWidth: .infinity, maxHeight: .infinity)
+          
+        case .loaded:
+          List {
+            ForEach(viewModel.rows, id: \.id) { row in
+              HardcoverBookRow(viewModel: row)
+                .onTapGesture {
+                  viewModel.onRowTapped(row)
+                  dismiss()
+                }
+            }
+          }
+          .listStyle(.plain)
+
+        case .error(let message):
+          VStack(spacing: 16) {
+            Image(systemName: "exclamationmark.triangle")
+              .font(.system(size: 48))
+              .foregroundColor(.red)
+              .accessibilityHidden(true)
+            
+            Text("hardcover_error_title".localized)
+              .font(.headline)
+            
+            Text(message)
+              .foregroundColor(.secondary)
+              .multilineTextAlignment(.center)
+              .padding(.horizontal)
+          }
+          .frame(maxWidth: .infinity, maxHeight: .infinity)
+        }
+      }
+    .onAppear(perform: viewModel.onAppear)
+    .toolbar {
+        ToolbarItem(placement: .navigationBarTrailing) {
+            Button("hardcover_unlink_button".localized) {
+              viewModel.onUnlinkTapped()
+              dismiss()
+            }
+        }
+    }
```

Comment:
```suggestion
    }
    .tint(themeViewModel.linkColor)
```

let's add the `@StateObject var themeViewModel = ThemeViewModel()`